packages/x-data-grid/src/components/cell/GridCell.tsx

```diff
@@ -7,8 +7,10 @@ import {
   unstable_ownerDocument as ownerDocument,
   unstable_capitalize as capitalize,
 } from '@mui/utils';
+import { tooltipClasses, TooltipProps } from '@mui/material/Tooltip';
```

Comment:
Do you see any way to do this without adding material imports here? I need to get rid of these for the design-system agnostic OKR.

Reply:
This PR is not explicitly requested by anyone. I just saw that we are using different approach for the header row cell and content row cell tooltips.
If it creates too much trouble, the PR can be discarded.
What will be the replacement for the current header cell tooltip?